app/models/user.rb

```diff
@@ -689,6 +689,12 @@ def badge_count
   def featured_user_badges(limit=3)
     user_badges
         .group(:badge_id)
+        .where("badge_id >= (
+                  SELECT MAX(ub2.badge_id)
+                    FROM user_badges ub2
+                   WHERE ub2.badge_id IN (#{Badge.trust_level_badge_ids.join(",")})
```

Comment:
.where("id in :ids or thing = :thing", ids: [1,2,3], thing: "thing") ...sometimes in select we have to work around this stuff, but in general try to be safe.